versioned_docs/version-v1.10.0/distributed-cluster-deployment-guide/01-distributed-cluster-deployment-guide-doc.md

```diff
@@ -0,0 +1,22 @@
+## Distributed Consistency Protocol for EventMesh
+- **Current State:**
+  - Kafka utilizes KRaft for in-sync replication among brokers.
+  - Event-mesh does not fully implement JRaft.
+  - A partially functional JRaft PR has been merged, but it requires a fixed cluster list defined in the `eventmesh.properties` file (immutable).
+  - This limits elastic scaling of EventMesh cluster nodes.
+- **JRaft Potential:**
+  - JRaft allows for elastic scaling by dynamically discovering new clusters.
+  - Nodes can be added by specifying IP and port in `eventmesh.properties`.
+  - Leader election occurs if the leading node fails.
+  - Recovered leading nodes become followers.
+- **Impact on Registry Center:**
+  - Implementing JRaft eliminates the need for Nacos as a registry center.
+  - Existing SDK pattern relies on static IP insertion in `eventmesh.properties`.
+  - Node failure results in service failure.
```

Comment:
Line 14 confuses me; it describes the current state of JRaft, which is unrelated to the section titled "Impact on Registry Center".